src/webots/editor/WbProjectRelocationDialog.cpp

```diff
@@ -280,11 +282,10 @@ int WbProjectRelocationDialog::copyWorldFiles() {
   // copy the .wbt file, the .wbproj file
   QDir targetPathDir(mTargetPath + "/worlds");
   targetPathDir.mkpath(".");
-  const QString &targetWorld(mTargetPath + "/worlds/" + worldFileBaseName);
-  WbProtoManager::instance()->updateCurrentWorld(targetWorld);
-  if (QFile::copy(mProject->worldsPath() + worldFileBaseName + ".wbt", targetWorld + ".wbt")) {
-    QFile::setPermissions(targetWorld + ".wbt",
-                          QFile::permissions(targetWorld + ".wbt") | QFile::WriteOwner | QFile::WriteUser);
+  const QString &mTargetWorld(mTargetPath + "/worlds/" + worldFileBaseName);
```

Comment:
I am not sure the dereference is useful here:
```suggestion
  const QString mTargetWorld(mTargetPath + "/worlds/" + worldFileBaseName);
```